demo/body3d_pose_lifter_demo.py

```diff
@@ -260,6 +260,7 @@ def get_pose_lift_results(args, visualizer, pose_lifter, pose_est_results_list,
 
     pred_3d_data_samples = merge_data_samples(pose_lift_results)
     det_data_sample = merge_data_samples(pose_est_results)
+    pred_3d_pred = pred_3d_data_samples.get('pred_instances', None)
```

Comment:
```suggestion
    pred_3d_instances = pred_3d_data_samples.get('pred_instances', None)
```
What about using `pred_3d_instances`? It seems to be easier to understand.